### PR TITLE
Translate: Advanced Guides > Refs and the DOM

### PR DESCRIPTION
**Pull request by Darking360**
**Translation of Advanced Guides: Refs and the DOM**

**Related:** [Original issue: Spanish Translation Progress](https://github.com/reactjs/es.reactjs.org/issues/4)

**To Do:**

- [x] Finish first translation draft

- [x] Check translation

- [x] Check grammar and orthography

- [x] Submit final

### CODE REVIEW

**Comment by Darking360** [comment]
Hey guys! Quick question, the code examples should they also be translated? Or only the comments inside them?  🤔 

**CC:** @dmoralesm @carburo @alejandronanez

**Comment by alejandronanez** [comment]
Hey @Darking360, thanks for asking.

You should translate code comments, that's it - everything else, (variable names, JSX, component names, props) should remain untouched!

**Comment by Darking360** [comment]
Greetings guys! :muscle: 

I just finished the translation, would you please check it out? Let me know for any changes :pray: :rocket: 

**CC:** @alejandronanez @carburo @dmoralesm

File: content/docs/refs-and-the-dom.md [hunk 1]
```diff
@@ -251,9 +251,9 @@ class CustomTextInput extends React.Component {
 }
 ```
 
-React will call the `ref` callback with the DOM element when the component mounts, and call it with `null` when it unmounts. Refs are guaranteed to be up-to-date before `componentDidMount` or `componentDidUpdate` fires.
+React llamara al callback del `ref` con el elemento del DOM cuando el componente sea montado, y lo llamara con `null` cuando este se desmonte. Se asegura que las referencias serán actualizadas antes que el `componentDidMount` o el `componentDidUpdate` sean ejecutados.
 
-You can pass callback refs between components like you can with object refs that were created with `React.createRef()`.
+Puedes pasar una referencia mediante callback entre componentes tal como puedes con los objetos de referencias creados con `React.createRef()`
```
Comment (alejandronanez) [inline content/docs/refs-and-the-dom.md]:
```suggestion
Puedes pasar una referencia mediante callback entre componentes tal como puedes con los objetos de referencias creados con `React.createRef().`
```

**Comment by Darking360** [comment]
Greetings @alejandronanez :sun_with_face: 

I just updated the PR and add the missing dot, if you could take a look at it would be great :muscle: Let me know if anything changes.

**CC:** @carburo @dmoralesm

**Pull request merged**